tictactoe/adapter-vdm/src/main/vdm/XO.vdmsl

```diff
@@ -68,8 +68,10 @@ functions
 			exists line in set winningLines &
 				line subset moves;
 
-  hasLost: Game * Player -> bool
-  hasLost(g, p) == not hasWon(g, p);
+
+    hasLost: Game * Player -> bool
+    hasLost(g, p) == not hasWon(g, p);
+
 
 	whoWon: Game -> Player
 	whoWon(g) ==
```

Comment:
I was trying to rationalise the whitespace here but maybe I went in the wrong direction.